src/jenkins-enhanced.coffee

```diff
@@ -353,13 +486,86 @@ class HubotJenkinsPlugin extends HubotMessenger
     "HEALTH: #{result}\n"
 
   _getJob: (escape = false) =>
-    job = @msg.match[1].trim()
-
-    # if the provided name is an alias, provide it's mapped job name
-    aliases = @_getSavedAliases()
-    job     = aliases[job] if aliases[job]
-
-    if escape then @_querystring.escape(job) else job
+    # check if its a job we already stored
+    if (typeof(@msg.match[1]) == "object")
+      return @msg.match[1]
+    # check if the user gave us a folder path to follow and a job
+    else
```

Comment:
Being the initial `if` expression does a hard return, this `else` can be removed and code following can be unindented to reduce nesting.